packages/launchpad/src/global/GlobalPage.cy.tsx

```diff
@@ -68,6 +68,13 @@ describe('<GlobalPage />', { viewportHeight: 900, viewportWidth: 1200 }, () => {
     it('renders projects', () => {
       cy.findByText(testProject).should('be.visible')
       cy.findByText(testProjectPath).should('be.visible')
+      // validate that there is some padding on the bottom
+      cy.get('[data-cy="project-card"]')
+      .parent()
+      .invoke('outerHeight')
+      .then((parentHeight) => {
+        cy.get('[data-cy="project-card"]').invoke('outerHeight').should('be.lt', parentHeight)
```

Comment:
Looks like we don't have any Percy snapshots for this global mode project list, let's add one here.

```suggestion
        cy.get('[data-cy="project-card"]').invoke('outerHeight').should('be.lt', parentHeight)
        cy.percySnapshot()
```